src/components/PoolPositionCard/PoolPositionCard.tsx

```diff
@@ -108,7 +108,7 @@ const PoolPositionCard: React.FC<{ pair: Pair }> = ({ pair }) => {
       </Box>
 
       {showMore && <PoolPositionCardDetails pair={pair} />}
-      {stakingInfo && apyWithFee && (
+      {stakingInfo && !stakingInfo.ended && apyWithFee && (
```

Comment:
The code is unclear here.  What does pools have to do with staking info ending.  I can understand that for farms or syrup but why pools.

Please respond to my comment below

Reply:
We are showing this part if the farm is existing with the same pair of the pool. The fix is not to show this section for the ended farms.
<img width="688" alt="screenshot" src="https://user-images.githubusercontent.com/47202087/153968837-2cb4354c-2eb3-4fcc-b988-20212c59bf97.png">

Reply:
<img width="688" alt="screenshot" src="https://user-images.githubusercontent.com/47202087/153971369-2af8ba6a-2043-464f-acc4-578cc71b6c44.png">
This is to show APY when the farm with same pair of the pool is existing. This update is not to show the section for the ended farm.